.storybook/stories/dropdown/dropdown-relative-position.stories.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016-2025 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrDropdown, ClrDropdownModule } from '@clr/angular';
+import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
+
+import { CommonModules } from '../../helpers/common';
+
+export default {
+  title: 'Dropdown/Dropdown Relative Position',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrDropdownModule],
+    }),
+  ],
+  component: ClrDropdown,
+};
+
+const DropdownRelativePositionTemplate: StoryFn = args => ({
+  template: `
+    <div class="content-area" style="position: relative">
+      <div style="margin-top: 800px"></div>
```

Comment:
Reduce the `top-margin` to make the dropdown visible.

```suggestion
      <div style="margin-top: 400px"></div>
```